### ISSUE TITLE
[PD] What do you think?

### ISSUE THREAD

**Issue by aishaathmanlali** [aishaathmanlali/Coursework-Planner#81]
From Module-JS1 created by [kfklein15](https://github.com/kfklein15): CodeYourFuture/Module-JS1#41  

### Coursework content
Use critical thinking skills to answer a client’s request on a web page you developed.

### Estimated time in hours

1

### What is the purpose of this assignment?
A client requested the navigation bar at the bottom of their web page. Research what best practice is and why this would be a good, a bad or a neutral idea.

These are the tips to help you write it:
- Provide appropriate and sufficient arguments and examples
- Research arguments against and for the statement before coming to your conclusions.
- Choose terms that are precise, appropriate, and persuasive
- Make clear the transitions from one thought to another to ensure the overall logic of the presentation

### How to submit
- Create a text with at least 250 words.
- Make sure your text is reviewed by a grammar tool. It should have at most 3 mistakes.
- Add the link to the Google doc to comment on this issue. Make sure it is open for anyone to comment.

**Comment by aishaathmanlali** [comment]
https://docs.google.com/document/d/1ug27I4TBDuWkQwytgabLnDnZYDZSr5iZFuVl-3VTu3A/edit?usp=sharing

**Comment by kroker42** [comment]
@aishaathmanlali 

This is a nice text, it is clear that you have put effort into your arguments for and against. It is well written and flows well.

The AI detector says there is a near 60% chance that it is AI generated. 
Try not to rely on AI too much. Make sure to express your own opinions in your own words, it creates a stronger, more authentic document.

I've added a few suggestions for consideration in the document - they are for your consideration and optional.

To complete the assignment:
Please add your recommendation. You have made some strong arguments for and against - what is your opinion on the best way forward?

**Comment by aishaathmanlali** [comment]
@kroker42 Thank you for the feedback. However, I cannot choose the best one because I believe each way can be best according to the purpose of the website

**Comment by kroker42** [comment]
Fair enough, what I’m after is a clearer conclusion at the end - the conclusion can simply be that both solutions are equally valid and you need more information to choose. 
As-is, it’s a bit open ended, but it’s not a big deal.
